### ISSUE TITLE
[headings] Headings for aside use incorrect level

### ISSUE THREAD

**Issue by UsmanAfzal** [WeAreSnook/OR-UK-frontend#160]
URL: https://openreferraluk.org/standard-community

The heading "On this page" within the aside is using a h4 when semantically should be a h2.

<img width="409" alt="Screenshot 2021-06-22 at 21 05 47" src="https://user-images.githubusercontent.com/2459151/122992350-ed548e80-d39d-11eb-82a3-8e5b3004fdbb.png">

**Comment by UsmanAfzal** [comment]
Checked and approved
